app/controllers/devise_token_auth/confirmations_controller.rb

```diff
@@ -6,21 +6,23 @@ def show
       @resource = resource_class.confirm_by_token(params[:confirmation_token])
 
       if @resource.persisted?
-        expiry = nil
+        lifespan = nil
```

Comment:
don't need to init these variables like js, ruby will take it from the scope

Reply:
Good remark.